### ISSUE TITLE
Performance evaluation

### ISSUE THREAD

**Issue by paulafernalia** [Slowika/Bag-of-Recipes#6]
Possible idea: take test set created from training set, hide 1 ingredient, predict rating for that ingredient.
